console/src/app/pages/orgs/org-list/org-list.component.ts

```diff
@@ -0,0 +1,99 @@
+import { SelectionModel } from '@angular/cdk/collections';
+import { AfterViewInit, Component, ViewChild } from '@angular/core';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatSort } from '@angular/material/sort';
+import { MatTableDataSource } from '@angular/material/table';
+import { Router } from '@angular/router';
+import { BehaviorSubject, from, Observable, of } from 'rxjs';
+import { catchError, finalize, map } from 'rxjs/operators';
+import { enterAnimations } from 'src/app/animations';
+import { MyProjectOrgSearchKey, MyProjectOrgSearchQuery, Org, SearchMethod } from 'src/app/proto/generated/auth_pb';
+import { GrpcAuthService } from 'src/app/services/grpc-auth.service';
+
+@Component({
+    selector: 'app-org-list',
+    templateUrl: './org-list.component.html',
+    styleUrls: ['./org-list.component.scss'],
+    animations: [
+        enterAnimations,
+    ],
+})
+export class OrgListComponent implements AfterViewInit {
+    public orgSearchKey: MyProjectOrgSearchKey | undefined = undefined;
+
+    @ViewChild(MatPaginator) public paginator!: MatPaginator;
+    @ViewChild(MatSort) sort!: MatSort;
+
+    public dataSource!: MatTableDataSource<Org.AsObject>;
```

Comment:
Shouldnt this be OrgView instead of Org?

Reply:
SearchMyProjectOrgs returns an Org list. Should therefore be right.